modules/aws_account_permissions/main.tf

```
@@ -0,0 +1,208 @@
+terraform {
+  required_providers {
+    illumio-cloudsecure = {
+      source  = "illumio/illumio-cloudsecure"
+      version = "~> 1.0.10"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.0"
+    }
+  }
+}
+data "aws_organizations_organization" "current" {}
+
+data "aws_partition" "current" {}
+
+data "aws_caller_identity" "current" {}
+
+resource "random_uuid" "role_secret" {}
+
+resource "aws_iam_role_policy" "IllumioCloudAWSIntegrationPolicy" {
+  name = "IllumioCloudAWSIntegrationPolicy"
```

1. Don't hardcode names.
2. Define tags.
```suggestion
  name = var.iam_name_prefix + "Policy"
  tags = var.tags
```

---

I fixed this.